### PR TITLE
Wrap on words, not characters when possible.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1625" title="DX-1625" target="_blank">StoryDX-1625</a>  Help output should wrap on words, not characters
  </td></table>
  <br />
 

Note: the existing unit tests cover wrapping edge cases (no pun intended).

### CODE REVIEW

File: internal/colorize/cropped.go [hunk 1]
```diff
@@ -36,8 +36,33 @@ func GetCroppedText(text string, maxLen int, includeLineEnds bool) CroppedLines
...
+			if !isLineEnd && entry.Length == maxLen && !nextCharIsSpace {
+				// Put the current word on the next line, if possible.
+				// Find the start of the current word and its printed length, taking color ranges and
+				// multi-byte characters into account.
+				i := len(entry.Line) - 1
+				for ; i > 0; i-- {
+					if isSpace(entry.Line[i]) {
+						break
+					}
+					if !inRange(pos-(len(entry.Line)-i), colorCodes) && !isUTF8TrailingByte(entry.Line[i]) {
```
Comment (MDrakos) [inline internal/colorize/cropped.go:51]: nit: a comment here _might_ be helpful though it's fairly evident now that I've read it a few times. I'll leave it up to you.

**Review by MDrakos — APPROVED** [review]
Nice work!
